packages/shared/src/components/modals/streaks/FirstStreakModal.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { ReactElement } from 'react';
+import { LazyModalCommonProps, Modal } from '../common/Modal';
+import { DayStreak, Streak } from '../../streak/popup';
+import { IconSize } from '../../Icon';
+import { Button, ButtonVariant } from '../../buttons/ButtonV2';
+import CloseButton from '../../CloseButton';
+
+export default function FirstStreakModal({
+  onRequestClose,
+  ...props
+}: LazyModalCommonProps): ReactElement {
+  return (
+    <Modal
+      {...props}
+      onRequestClose={onRequestClose}
+      kind={Modal.Kind.FlexibleCenter}
+      size={Modal.Size.XSmall}
+      className="p-6"
+    >
+      <CloseButton className="right-2 top-2" onClick={onRequestClose} />
```

Comment:
Think you need absolute here as well with buttonV2 changes.
@kopancek ?

Reply:
Moved to ModalClose. It should have a default `absolute` value.